src/components/Text/index.tsx

```diff
@@ -0,0 +1,15 @@
+import styles from './styles.css';
+import classNames from 'classnames';
+import { ReactNode } from 'react';
+
+interface Props {
```

Comment:
maybe name could be `TextProps` ?

Reply:
I'm not sure it is necessary since we have only one component in this file.
I would say it is easier to read the code if every file contains the same interface named `Props`